README.md

```diff
@@ -102,19 +104,21 @@ os.environ["MODIN_ENGINE"] = "dask"  # Modin will use Dask
 import modin.pandas as pd
 ```
 
-Check [this Modin docs section](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html) for Omnisci engine setup.
+Check [this Modin docs section](https://modin.readthedocs.io/en/latest/developer/using_omnisci.html) for Omnisci engine setup.
```

Comment:
The note below should be corrected.
from
```
**Note: You should not change the engine after you have imported Modin as it will result in undefined behavior**
```
to
```
**Note: You should not change the engine after your first operation with Modin as it will result in undefined behavior**
```